Sources/Public/Animation/LottieAnimationViewInitializers.swift

```diff
@@ -198,3 +328,12 @@ extension LottieAnimationView {
 enum LottieDownloadError: Error {
   case downloadFailed
 }
+
+// MARK: - Foundation.Bundle + Sendable
+
+/// Necessary to suppress warnings like:
+/// ```
+/// Non-sendable type 'Bundle' exiting main actor-isolated context in call to non-isolated
+/// static method 'named(_:bundle:subdirectory:dotLottieCache:)' cannot cross actor boundary
+/// ```
+extension Foundation.Bundle: @unchecked Sendable { }
```

Comment:
<img width="489" alt="image" src="https://user-images.githubusercontent.com/1811727/201428554-d327d48c-169e-47d4-afdf-7d1a9fdb8eef.png"> 

😛

Reply:
Would you be able to add a note explaining that this conformance doesn't conflict with conformances defined in other modules? As-is this would be a concern but seems like it's not an issue

Reply:
Since `Sendable` is a marker protocol without any runtime component, this doesn't cause conflicts with other modules (I checked that multiple modules in the same graph can define this extension). Will mention this in the comment.